### ISSUE TITLE
Is it a new firmware or an old firmware?

### ISSUE THREAD

**Issue by fribse** [EUROTRONIC-Technology/Spirit-ZigBee#9]
Dear @Witriol 
I've successfully changed the firmware on my thermostats to the one released here.
I'm a but baffled that it has an older date code than the one I had. So is this just an old firmware you've released? Are you working on a new firmware, or why is the datecode older?

**Comment by magicdude4eva** [comment]
I also would appreciate info too - to me it looks like the old firmware - see my current version:
![image](https://user-images.githubusercontent.com/1632781/106361677-78fe4d80-631f-11eb-8993-dd6a437ffece.png)

**Comment by Witriol** [comment]
@fribse Yes, it is an old firmware that I have received from Eurotronic to release. It is dated before problematic 20191014 firmware which came in two versions. Currently I'm waiting for another instructions from Eurotronic and answers for some community questions as well.

@magicdude4eva Thank you for your pull request with guide for Home Assistant OTA update.

**Comment by fribse** [comment]
@Witriol Thankyou for a clear answer :-)

**Comment by MXMLN-sec** [comment]
Hi @Witriol 
can you provide us an update of the current situation? Have you received a newer version?

Thank you and best

**Comment by Witriol** [comment]
Hi @Ma-Ke I have no update from Eurotronic and I think I will not receive one based on previous events, unfortunately.
